pkg/api/errors/errors.go

```diff
@@ -92,6 +92,14 @@ func FromObject(obj runtime.Object) error {
 	return &UnexpectedObjectError{obj}
 }
 
+// TypeNotFoundError is returned when specified type
+// can not found in schema
+type TypeNotFoundError string
```

Comment:
This file contains api errors generated by the server.
I dont think that this is the right place to define this error (which is used only by kubectl).

I think it is better to move it to pkg/api/validation/schema.go